src/resources/js/v2/tickets-commerce.js

```diff
@@ -82,6 +88,174 @@ tribe.tickets.commerce = {};
 		$document.on( obj.customEvents.hideLoader, obj.loaderHide );
 	};
 
+	/**
+	 * Updates the total price displayed on the page.
+	 *
+	 * @since TBD
+	 * @param {string} newAmount The new total amount to display.
+	 */
+	obj.updateTotalPrice = function (newAmount) {
+		const $totalPriceElement = $( '.tribe-tickets__commerce-checkout-cart-footer-total-wrap' );
+
+		const parser = new DOMParser();
+		const unescapedAmount = parser.parseFromString(`<!doctype html><body>${newAmount}`, 'text/html').body.textContent;
+
+		$totalPriceElement.text(unescapedAmount);
+	};
+
+	obj.updateCouponDiscount = function (discount) {
+		const $couponValueElement = $( obj.selectors.couponAppliedValue );
+
+		// Use DOMParser to unescape the discount value
+		const parser = new DOMParser();
+		const unescapedDiscount = parser.parseFromString(`<!doctype html><body>${discount}`, 'text/html').body.textContent;
+
+		$couponValueElement.text(unescapedDiscount);
+	};
+
+
+	obj.bindCouponApply = function() {
+		let ajaxInProgress = false;
+
+		$document.on( 'click', obj.selectors.couponApplyButton, applyCoupon );
+		$document.on( 'keydown', obj.selectors.couponInput, function( e ) {
+			if ( e.key === 'Enter' ) {
+				e.preventDefault();
+				applyCoupon();
+			}
+		});
```

Comment:
Invert the logic, switch the variable name to `event` and use `keyCode` to avoid having OS differences around `Enter`.